manual/manual/Makefile

```diff
@@ -39,25 +39,35 @@ index:
 	  && makeindex manual.kwd.idx
 
 
-# libref/style.css is used as witness for the generation of the html stdlib
-# reference.
-html: htmlman/libref/style.css etex-files
+# libref/style.css and comilerlibref/style.css are used as witness
```

Comment:
An aside: I don't understand what the term "as a witness" means here.

Reply:
Make does not support multi-output targets, so one specific file is used to "witness" the fact that a command outputting many other files has run. (And its edition time is used to check whether to rerun the command, when dependencies are younger.)

Reply:
Perhaps say exactly that? as in:

> Make does not support multi-output rules, so one specific file must be used to "witness" the fact that a command creating many other files has run. In this case, libref/style.css and comilerlibref/style.css are witnesses.

Reply:
BTW, this "feature" of Make has annoyed me for years.

Reply:
I don't think it is the right place to document Make.